### ISSUE TITLE
[Enabler] [Documentation] Ansible version requirement for z/OS core collection

### ISSUE THREAD

**Issue by ScYang1991** [ansible-collections/ibm_zos_core#630]
### Documentation issue description

I am testing z/OS core collection 1.4.0 under below ansible environment.
```
seungcheolyang@scmacbook ansible % ansible --version
ansible [core 2.14.0]
  config file = /Users/seungcheolyang/Desktop/ansible/ansible.cfg
  configured module search path = ['/Users/seungcheolyang/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/ansible
  ansible collection location = /Users/seungcheolyang/.ansible/collections:/usr/share/ansible/collections
  executable location = /Library/Frameworks/Python.framework/Versions/3.11/bin/ansible
  python version = 3.11.0 (v3.11.0:deaf509e8f, Oct 24 2022, 14:43:23) [Clang 13.0.0 (clang-1300.0.29.30)] (/Library/Frameworks/Python.framework/Versions/3.11/bin/python3.11)
  jinja version = 3.1.2
  libyaml = True
```
When I run playbooks, it runs but prints below warning.
```
[WARNING]: Collection ibm.ibm_zos_core does not support Ansible version 2.14.0
```
However, according to the requirements document (https://ibm.github.io/z_ansible_collections_doc/requirements/requirements.html), Ansible version 2.14.0 should be supported.
```
Ansible version: 2.9 or later
```
Can you specify which ansible versions are actually supported for each versions of z/OS core collection?  
Thanks.

### Ansible version

```SHELL
ansible [core 2.14.0]
  config file = /Users/seungcheolyang/Desktop/ansible/ansible.cfg
  configured module search path = ['/Users/seungcheolyang/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/ansible
  ansible collection location = /Users/seungcheolyang/.ansible/collections:/usr/share/ansible/collections
  executable location = /Library/Frameworks/Python.framework/Versions/3.11/bin/ansible
  python version = 3.11.0 (v3.11.0:deaf509e8f, Oct 24 2022, 14:43:23) [Clang 13.0.0 (clang-1300.0.29.30)] (/Library/Frameworks/Python.framework/Versions/3.11/bin/python3.11)
  jinja version = 3.1.2
  libyaml = True
```

### IBM z/OS Ansible core Version

v1.4.0-beta.1

**Comment by ddimatos** [comment]
Related #539

**Comment by ddimatos** [comment]
@ScYang1991 - all the collections available under [Red Hat® Ansible® [Certified Content](https://access.redhat.com/articles/3642632) for IBM Z®](https://ibm.github.io/z_ansible_collections_doc/index.html) are available as certified content through Automation hub and also hosted on Galaxy. Our offering including the `ibm_zos_core` collection are required to remain certified which are outlined in [AAP Life Cycle](https://access.redhat.com/support/policy/updates/ansible-automation-platform). 

If you are an AAP (Ansible Automation Platform) customer you'll need to be using the versions outlined in the above links which is 2.9 - 2.14. 

If you are pulling content from Galaxy the same should apply but we understand that users might often be using the latest so this quarter we have update our pipelines to include the development branch which is 2.15. 

When we released version 1.4.0 we had certified/tested the collection 2.9 & 2.11, subsequent releases will be tested through 2.15 (latest) which includes the upcoming [v1.5.0-beta.1](https://github.com/ansible-collections/ibm_zos_core/issues/433). 

The warning you saw can be ignored if your not planning to exercise AAP, there are ways to disable that warning noted in issue #539. I will update the documentation as it does say 2.9 or later, for 1.4.0 its been tested up through 2.11, but we are always willing to help with issues outside the scope of requirements. 

In addition to the links I provided you can always look at the meta data [here](https://github.com/ansible-collections/ibm_zos_core/blob/dev/meta/runtime.yml) to see what each version is supporting. 

For reference, where we had to adjust code for versions ansible 2.9, ansible-cre 2.10 and ansible-core 2.11 , since then the engine has stabilized and we have not seen any impact, hopefully this continues. 

In summary, you should be on ansible core 2.9 or ansible-core 2.11 for collection `ibm_zos_core` version 1.4.0. Should you be using another version of ansible-core and encounter an issue, feel free to open an issue an we will look into it.

Ill close this ticket but feel free to reopen it, thank you.

**Comment by ddimatos** [comment]
Addressed from this work item even though its in a different repository, doc will go live this week and this is these are the changes to better depict the ansible versions needed for the offering as a whole.

> Control node
> 
> A control node is any machine with Ansible® installed. From the control node, you can run commands and playbooks from a laptop, desktop, or server. However, Windows® is not a supported controller for Ansible collections managing a z/OS node.
> 
> All IBM z/OS collections are either [Ansible Automation Platform Certified Content](https://access.redhat.com/articles/3642632) (AAP) or undergoing certification. Thus, the supported versions of Ansible align with the [AAP Life Cycle](https://access.redhat.com/support/policy/updates/ansible-automation-platform) which currently is Ansible versions 2.9 - 2.14. For support, the controller must use the supported version of Ansible and when applicable the collections dependencies. Review the [Ansible Automation Platform Certified Content](https://access.redhat.com/articles/3642632) to determine which versions of Ansible a collection is supporting.
> 
> IBM z/OS collections minimally require these versions of software, for additional supported versions review [Ansible Automation Platform Certified Content](https://access.redhat.com/articles/3642632).
> 
>[Ansible version](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html): 2.9
> 
>[Python](https://www.python.org/downloads): 2.7 or later
> 
>[OpenSSH](https://www.openssh.com/)
> 
> Modules which can have Ansible tasks delegated to a localhost need not have OpenSSH enabled. Such modules often rely on a REST API for automation. Refer to the collections listed in the Managed node section to review which collections leverage a REST endpoint.
